beacon-chain/core/blocks/payload.go

```diff
@@ -278,9 +308,13 @@ func GetBlockPayloadHash(blk interfaces.BeaconBlock) ([32]byte, error) {
 	if IsPreBellatrixVersion(blk.Version()) {
 		return payloadHash, nil
 	}
-	payload, err := blk.Body().ExecutionPayload()
-	if err != nil {
+	payload, err := blk.Body().Execution()
+	switch {
+	case errors.Is(err, wrapper.ErrUnsupportedField):
```

Comment:
Why do we need this case? Anything before Bellatrix version should support `Execution()`